### DISCUSSION TITLE
How to ensemble predictions from ExtraSurvivalTrees models?

### DISCUSSION THREAD

**Discussion by ghost** [sebp/scikit-survival#387] [Q&A]
What would be the recommended way to ensemble the predictions from several ExtraSurivalTree models?
Thank you very much!

**Comment by sebp** [comment]
[sksurv.ensemble.ExtraSurvivalTrees](https://scikit-survival.readthedocs.io/en/stable/api/generated/sksurv.ensemble.ExtraSurvivalTrees.html) is an ensemble of sksurv.tree.SurvivalTree and does aggregate predictions of individual trees automatically when calling `predict`, `predict_survival_function`, or `predict_cumulative_hazard_function`.

**Reply by ghost** [reply]
Thanks a lot for your quick reply. I was more thinking of a scenario where we train several ExtraSurvivalTree models within a cross validation. Let's assume that we have 5 ExtraSurvialTree models and use them to predict on the test set. What would be the recommened way to ensemble the 5 test set prediction? For example,  (a) Average the risk scores from predict? (b) Average the survival curves from predict_survival_function, etc..
